src/components/Selfie/Intro.js

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react'
+import { h, Component } from 'preact'
+import PageTitle from '../PageTitle'
+import style from './style.css'
+import theme from '../Theme/style.css'
+import classNames from 'classnames'
+import Button from '../Button'
+import { localised } from '../../locales'
+import { trackComponent } from '../../Tracker'
+import withCrossDeviceWhenNoCamera from '../Capture/withCrossDeviceWhenNoCamera'
+import withCameraDetection from '../Capture/withCameraDetection'
+import { compose } from '~utils/func'
+
+const InstructionPure = ({instruction}) => (
+    <li key={instruction.key} className={style.introBullet}>
+      <span className={classNames(style.introIcon, style[`${instruction.key}Icon`])} />
+      <span className={classNames(style.bolder, style.introText)}>{instruction.text}</span>
+    </li>
+)
+
+const Instructions = ({listScreenReaderText, instructions}) => (
```

Comment:
this is pure as well, might make sense to add `Pure` to it as well, if you want to be consistent in this file

Reply:
changed